appinventor/docs/markdown/reference/components/experimental.md

```diff
@@ -8,7 +8,66 @@ title: Experimental
 
 Table of Contents:
 
+* [ChatBot](#ChatBot)
 * [FirebaseDB](#FirebaseDB)
+* [ImageBot](#ImageBot)
+
+## ChatBot  {#ChatBot}
+
+The ChatBot component is a non-visible component for chatting with an AI
+ chatbot. This version uses a proxy run by MIT that in turn uses the ChatGPT
+ generative large language model.
+
+
+
+### Properties  {#ChatBot-Properties}
+
+{:.properties}
+
+{:id="ChatBot.ApiKey" .text .ro .bo} *ApiKey*
+: Property for ApiKey
+
+{:id="ChatBot.Apikey" .text .wo .bo} *Apikey*
+: An ApiKey for ChatGPT. User supplied. If provided, we will use it instead of the
+ API key embedded in the chat proxy service.
+
```

Comment:
Since the Apikey was corrected to ApiKey, rebuilding the documentation with `ant docs` should consolidate these entries.